resources/public/api/conf/1.0/application.raml

```diff
@@ -294,15 +294,261 @@ uses:
                    LSANI-C1-residences or LSANI-C4-residences or NICTSEJO-C4-residences or LAA-C4-contact-details or HMCTS-C4-contact-details
       body:
         application/json:
-          schema: !include schemas/create-contact-details.json
-          example: !include examples/create-contact-details.json
+          schema: !include schemas/individuals/create-contact-details.json
+          example: !include examples/individuals/create-contact-details.json
       responses:
         201:
           description: The created contact details
           body:
             application/json:
-              example: !include examples/create-contact-details-response.json
-              type: !include schemas/create-contact-details-response.json
+              example: !include examples/individuals/create-contact-details-response.json
+              type: !include schemas/individuals/create-contact-details-response.json
+        400:
+          body:
+            application/json:
+              type: types.errorResponse
+              examples:
+                BadRequest:
+                  description: Invalid request
+                  value:
+                    code: INVALID_REQUEST
+
+  /organisations/self-assessment/{utr}/return/details:
+    uriParameters:
+      utr:
+        description: "A Unique Tax Reference"
+        type: string
+        required: true
+        example: "1234567890"
+    post:
+      displayName: "Create a self-assessment organisations payload"
```

Comment:
suggest `Create a self-assessment return details payload`? 

More applicable to the endpoint description?